### ISSUE TITLE
`reportMissingImports` underline while module actually resolved

### ISSUE THREAD

**Issue by Morphlng** [microsoft/vscode-jupyter#12174]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

## Description

I've recently came to this bug where each import statement in the `.ipynb` file is yellow underlined. I've already verified the installation of the package (e.g., numpy), and switched the conda env the notebook uses, the warning still won't go away.

The below picture shows you even more detail:
![vscode bug](https://user-images.githubusercontent.com/50166287/204069805-e94e1945-5dde-4c30-9038-09fbceeebbcf.png)

As you can see, the module name and package description has already shown, yet the warning came along with it. I believe this is a recent update bug since I didn't meet this before.

## Platform

1. Ubuntu 22.04.
2. Latest version of vscode (already reinstalled).
3. Latest version of jupyter plugin.

### VS Code Version

1.73.1

### Jupyter Extension Version

v2022.9.1303220346

### Coding Language and Runtime Version

Python v3.8.13

**Comment by DonJayamanne** [comment]
Thanks for filing this issue
Please could you:
* install vscode insiders
* Install the pre-release version of the jupyter extension

I believe the above steps will resolve the issue, if yes then please wait for the next release of vscode and the extension.

**Comment by Morphlng** [comment]
> * install vscode insiders
> * Install the pre-release version of the jupyter extension

I'm not convenient to install vscode insiders right now (sorry about that), but I've tried pre-release version of jupyter extension, and it doesn't help.

Also, I've found that this problem is not 100% triggered. I copied the whole workplace directory to another folder and then opened with vscode, it works fine. But even with the warning gone, it still seems to perform incorrect by showing each package information twice. (e.g., `(module) torch` will show in two lines, I'll post a picture later if possible)
